package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@reportportal/reportportal-agent-jest",
+  "name": "@reportportal/agent-js-jest",
```

Comment:
Please remove redundant `example:test` from scripts section.
Also `"url": "git+https://github.com/reportportal/reportportal-agent-jest.git"` isn't valid url now.
Should we specify author of package?